src/components/Wallet/WalletGenerator.jsx

```diff
@@ -129,22 +295,29 @@ class WalletGenerator extends React.Component {
 
   toggleImporters = (event) => {
     event.preventDefault();
-    const { setImportersVisible, configuring } = this.props;
+    const { setImportersVisible, configuring, resetWallet } = this.props;
+    
+    if (!configuring) {
+      this.setState({ generating: false }, () => {
+        resetWallet()
```

Comment:
Needed to reset the whole view if the wallet config was cleared. There was no way to do this before without refreshing. This makes for a nicer, smoother UI.